src/layouts/sidebar-sidecar/types.ts

```diff
@@ -17,6 +17,7 @@ interface BaseProps {
 	AlternateSidebar?: (props: any) => ReactElement
 	optInOutSlot?: ReactElement
 	versions?: VersionSelectItem[]
+	initialOpenMobileAuthControls?: boolean
```

Comment:
I don't love this drilling but I couldn't think of a different way to enable this opened menu state. Seemed like the easiest option with the current setup.